Dockerfile.dev

```diff
@@ -21,7 +21,11 @@ RUN apt-get update && \
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 
-RUN pip install poetry
+RUN pip install poetry pytest-playwright
+
+# Playwright dependencies to run browsers
+RUN playwright install
```

Comment:
Shouldn't these go after the `poetry install` so that we get the versions that are pinned in the `poetry.lock` file? Is there an equivalent `playwritght` type lockfile